### PR TITLE
Fixes #24157 - undefined SubscriptionsPage `tasks`

### PR DESCRIPTION
**Pull request by sharvit**
SubscriptionsPage `tasks` props is undefined
even though it has defaultProps and mapStateToProps.

### CODE REVIEW

File: webpack/scenes/Subscriptions/SubscriptionsPage.js [hunk 1]
```diff
@@ -38,36 +38,39 @@ class SubscriptionsPage extends Component {
   }
 
   static getDerivedStateFromProps(nextProps, prevState) {
-    const nextTaskId = nextProps.tasks[0] && nextProps.tasks[0].id;
+    const { tasks = [] } = nextProps;
```
Comment (amirfefer) [inline webpack/scenes/Subscriptions/SubscriptionsPage.js]: If you'll remove this empty array default value, will it be undefined?  especially when an empty array is defined as the default state? seems very odd to me.

Comment (sharvit) [reply webpack/scenes/Subscriptions/SubscriptionsPage.js]: Somehow it is happen when I run `centos7-katello-bats-ci` but I couldn't reproduce it locally.

Comment (sharvit) [reply webpack/scenes/Subscriptions/SubscriptionsPage.js]: Same problem with `centos7-katello-nightly`

**Comment by waldenraines** [comment]
@sharvit looks like there is a test to fix.

**Comment by sharvit** [comment]
Thanks @waldenraines, fixed.

**Comment by waldenraines** [comment]
[test katello]

**Comment by sharvit** [comment]
It did fixed the nightly issue! Thanks @waldenraines :+1:

**Comment by waldenraines** [comment]
Nice, thank you @sharvit!

**Pull request merged**